_docs_v7/Solver-Setup.md

```diff
@@ -40,15 +49,111 @@ SU2 is capable of dealing with different kinds of physical problems. The kind of
 |`FEM_NAVIER_STOKES`| **Navier-Stokes' equation** | Discontinuous Galerkin FEM |
 |`MULTIPHYSICS` | Multi-zone problem with different solvers in each zone | - |
 
+### Turbulence modeling ###
+
+This section describes how to define the turbulence model in SU2 to be coupled to RANS or INC_RANS. The selected model must be specified in the config file by the `KIND_TURB_MODEL` option. The current available turbulence models are the:
+ - `NONE`: No turbulence model.
+ - `SST`: Mentor Shear Stress Transport.
+ - `SA`: Spalart-Allmaras.
+ 
+Different corrections and variations are implemented for each turbulence model. These are specified accordingly either in `SST_OPTIONS` or in `SA_OPTIONS`. The folowing options are available:
+- `SST_OPTIONS`:
+  - `NONE`: No SST turbulence model (default).
+  - `V1994m`: refers to the "Standard" Menter SST model from 1994 with the modified constant $\sigma_{k1}$ and the redefinition of the turbulent eddy viscosity.
+  - `V2003m`: refers to the Menter SST model from 2003 ignoring the $\tau_{ij}$ in the momentum and energy equations and the production term is approximated by $P = \mu_t S^2$.
+  - `VORTICITY`: refers to the vorticity source term correction.
+  - `KATO_LAUNDER`: refers to the Kato-Launder correction.
+  - `UQ`: refers to the V1994m version with uncertainty quantification modifications.??
+  - `SUSTAINING`: refers to the controlled decay correction.
+
+- `SA_OPTIONS`:
+  - `NONE`: refers to the modification where the $f_{t2}$ term is set to zero, i.e., $c_{t3} = 0$ (default).
```

Comment:
This does not have to be repeated here... just point to the section in Physical definition.